diff --git a/src/postorius/templates/postorius/index.html b/src/postorius/templates/postorius/index.html
index 0326b49..1d19194 100644
--- a/src/postorius/templates/postorius/index.html
+++ b/src/postorius/templates/postorius/index.html
@@ -58,10 +58,10 @@
             </ul>
           </div>
     {% endif %}
-    {% if user.is_superuser %}
+    {% if user.is_authenticated %}
       <div class="col">
         <p class="text-right">
-            {% if domain_count < 1 %}
+            {% if domain_count < 1 and user.is_superuser %}
                 <a class="btn btn-success" href="{% url 'domain_new' %}">{% trans 'Create New Domain' %}</a>
             {% else %}
                 <a class="btn btn-success" href="{% url 'list_new' %}">{% trans 'Create New List' %}</a>
diff --git a/src/postorius/tests/mailman_api_tests/test_list_new.py b/src/postorius/tests/mailman_api_tests/test_list_new.py
index 4581412..be39a4b 100644
--- a/src/postorius/tests/mailman_api_tests/test_list_new.py
+++ b/src/postorius/tests/mailman_api_tests/test_list_new.py
@@ -40,7 +40,6 @@ class ListCreationTest(ViewTestCase):
         self.domain = self.mm_client.create_domain('example.com')
 
     def test_permission_denied(self):
-        self.client.login(username='user', password='pwd')
         response = self.client.get(reverse('list_new'))
         self.assertEqual(response.status_code, 403)
 
diff --git a/src/postorius/views/list.py b/src/postorius/views/list.py
index 85e0d78..4c43d12 100644
--- a/src/postorius/views/list.py
+++ b/src/postorius/views/list.py
@@ -54,7 +54,6 @@ from django_mailman3.signals import (
 from postorius.auth.decorators import (
     list_moderator_required,
     list_owner_required,
-    superuser_required,
 )
 from postorius.auth.mixins import ListOwnerMixin
 from postorius.forms import (
@@ -960,7 +959,6 @@ def _get_default_style():
 
 
 @login_required
-@superuser_required
 def list_new(request, template='postorius/lists/new.html'):
     """
     Add a new mailing list.
