diff --git a/dist/templates/about/about.html.hbs b/dist/templates/about/about.html.hbs
index 104a821..0b2f30c 100644
--- a/dist/templates/about/about.html.hbs
+++ b/dist/templates/about/about.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about">
-    <center><h2>About | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2>About | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

     <p>
         The <span class="brand">keys.openpgp.org</span> server is a public service for the
diff --git a/dist/templates/about/api.html.hbs b/dist/templates/about/api.html.hbs
index cc6b7e3..b639626 100644
--- a/dist/templates/about/api.html.hbs
+++ b/dist/templates/about/api.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

   <p>
     Hagrid implements both the legacy HKP interface, as well as our
diff --git a/dist/templates/about/faq.html.hbs b/dist/templates/about/faq.html.hbs
index 411fd8b..642695f 100644
--- a/dist/templates/about/faq.html.hbs
+++ b/dist/templates/about/faq.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | FAQ | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | FAQ | <a href="/about/privacy">Privacy</a></h2></center>

     <p>
         <strong>For instructions, see our <a href="/about/usage">usage guide</a>.</strong>
diff --git a/dist/templates/about/news.html.hbs b/dist/templates/about/news.html.hbs
index 34f996c..d0b319c 100644
--- a/dist/templates/about/news.html.hbs
+++ b/dist/templates/about/news.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about">
-    <center><h2><a href="/about">About</a> | News | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | News | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

     <h2 id="2023-04-28-governance">
       <div style="float: right; font-size: small; line-height: 2em;">2023-04-28 📅</div>
diff --git a/dist/templates/about/privacy.html.hbs b/dist/templates/about/privacy.html.hbs
index 56158cf..c2376d4 100644
--- a/dist/templates/about/privacy.html.hbs
+++ b/dist/templates/about/privacy.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | Privacy</h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | Privacy</h2></center>

     <h3>Name and contact details</h3>
     <p>
diff --git a/dist/templates/about/stats.html.hbs b/dist/templates/about/stats.html.hbs
index df3c728..3a12d27 100644
--- a/dist/templates/about/stats.html.hbs
+++ b/dist/templates/about/stats.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | Stats | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | Stats | <a href="/about/privacy">Privacy</a></h2></center>

     <h3>Verified email addresses</h3>

diff --git a/dist/templates/about/usage.html.hbs b/dist/templates/about/usage.html.hbs
index 2c395f1..9011f24 100644
--- a/dist/templates/about/usage.html.hbs
+++ b/dist/templates/about/usage.html.hbs
@@ -1,6 +1,6 @@
 {{#> layout }}
 <div class="about usage">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | Usage | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | Usage | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

     <p>
         On this page, we collect information on how to use
diff --git a/templates-translated/de/about/about.html.hbs b/templates-translated/de/about/about.html.hbs
index 184a93d..fbb026d 100644
--- a/templates-translated/de/about/about.html.hbs
+++ b/templates-translated/de/about/about.html.hbs
@@ -1,5 +1,5 @@
 <div class="about">
-    <center><h2>Übersicht | <a href="/about/news">News</a> | <a href="/about/usage">Nutzung</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Statistik</a> | <a href="/about/privacy">Privacy Policy</a>
+    <center><h2>Übersicht | <a href="/about/usage">Nutzung</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Statistik</a> | <a href="/about/privacy">Privacy Policy</a>
 </h2></center>

     <p>Der <span class="brand">keys.openpgp.org</span> Server ist ein öffentlicher Service für die Verteilung von OpenPGP-Schlüsseln, üblicherweise als "Keyserver" bezeichnet.</p>
diff --git a/templates-translated/de/about/faq.html.hbs b/templates-translated/de/about/faq.html.hbs
index 83eb0b7..4323e25 100644
--- a/templates-translated/de/about/faq.html.hbs
+++ b/templates-translated/de/about/faq.html.hbs
@@ -1,6 +1,6 @@
 <div class="about">
     <center><h2>
-<a href="/about">Übersicht</a> | <a href="/about/news">News</a> | <a href="/about/usage">Nutzung</a> | FAQ | <a href="/about/stats">Statistik</a> | <a href="/about/privacy">Privacy Policy</a>
+<a href="/about">Übersicht</a> | <a href="/about/usage">Nutzung</a> | FAQ | <a href="/about/stats">Statistik</a> | <a href="/about/privacy">Privacy Policy</a>
 </h2></center>

     <p><strong>Für Details zur Nutzung, siehe <a href="/about/usage">Nutzungshinweise</a>.</strong></p>
diff --git a/templates-translated/de/about/stats.html.hbs b/templates-translated/de/about/stats.html.hbs
index eb3e5ae..2cb6189 100644
--- a/templates-translated/de/about/stats.html.hbs
+++ b/templates-translated/de/about/stats.html.hbs
@@ -1,6 +1,6 @@
 <div class="about">
     <center><h2>
-<a href="/about">Übersicht</a> | <a href="/about/news">News</a> | <a href="/about/usage">Nutzung</a> | <a href="/about/faq">FAQ</a> | Statistik | <a href="/about/privacy">Privacy Policy</a>
+<a href="/about">Übersicht</a> | <a href="/about/usage">Nutzung</a> | <a href="/about/faq">FAQ</a> | Statistik | <a href="/about/privacy">Privacy Policy</a>
 </h2></center>

     <h3>Bestätigte Email-Adressen</h3>
diff --git a/templates-translated/de/about/usage.html.hbs b/templates-translated/de/about/usage.html.hbs
index 198f61d..5276595 100644
--- a/templates-translated/de/about/usage.html.hbs
+++ b/templates-translated/de/about/usage.html.hbs
@@ -1,6 +1,6 @@
 <div class="about usage">
     <center><h2>
-<a href="/about">Übersicht</a> | <a href="/about/news">News</a> | Nutzung | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Statistik</a> | <a href="/about/privacy">Privacy Policy</a>
+<a href="/about">Übersicht</a> | Nutzung | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Statistik</a> | <a href="/about/privacy">Privacy Policy</a>
 </h2></center>

     <p>Auf dieser Seite sammeln wir Anleitungen zur Nutzung von <span class="brand">keys.openpgp.org</span> mit unterschiedlichen OpenPGP-Anwendungen. Wir sind noch dabei, weitere Anleitungen hinzuzufügen - falls du eine bestimmte vermisst, lass es uns einfach wissen.</p>
diff --git a/templates-untranslated/about/about.html.hbs b/templates-untranslated/about/about.html.hbs
index d2b8fb0..9a86029 100644
--- a/templates-untranslated/about/about.html.hbs
+++ b/templates-untranslated/about/about.html.hbs
@@ -1,5 +1,5 @@
 <div class="about">
-    <center><h2>About | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2>About | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

     <p>
         The <span class="brand">keys.openpgp.org</span> server is a public service for the
diff --git a/templates-untranslated/about/faq.html.hbs b/templates-untranslated/about/faq.html.hbs
index 76e99f9..b1f2294 100644
--- a/templates-untranslated/about/faq.html.hbs
+++ b/templates-untranslated/about/faq.html.hbs
@@ -1,5 +1,5 @@
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | FAQ | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | FAQ | <a href="/about/privacy">Privacy</a></h2></center>

     <p>
         <strong>For instructions, see our <a href="/about/usage">usage guide</a>.</strong>
diff --git a/templates-untranslated/about/news.html.hbs b/templates-untranslated/about/news.html.hbs
index c9c10e5..cd5e87e 100644
--- a/templates-untranslated/about/news.html.hbs
+++ b/templates-untranslated/about/news.html.hbs
@@ -1,5 +1,5 @@
 <div class="about">
-    <center><h2><a href="/about">About</a> | News | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | News | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

     <h2 id="2023-04-28-governance">
       <div style="float: right; font-size: small; line-height: 2em;">2023-04-28 📅</div>
diff --git a/templates-untranslated/about/privacy.html.hbs b/templates-untranslated/about/privacy.html.hbs
index 415f1b5..4dd2d78 100644
--- a/templates-untranslated/about/privacy.html.hbs
+++ b/templates-untranslated/about/privacy.html.hbs
@@ -1,5 +1,5 @@
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | Privacy</h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | Privacy</h2></center>

     <h3>Name and contact details</h3>
     <p>
diff --git a/templates-untranslated/about/stats.html.hbs b/templates-untranslated/about/stats.html.hbs
index 89bd873..c71d3ed 100644
--- a/templates-untranslated/about/stats.html.hbs
+++ b/templates-untranslated/about/stats.html.hbs
@@ -1,5 +1,5 @@
 <div class="about">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | Stats | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | <a href="/about/usage">Usage</a> | <a href="/about/faq">FAQ</a> | Stats | <a href="/about/privacy">Privacy</a></h2></center>

     <h3>Verified email addresses</h3>

diff --git a/templates-untranslated/about/usage.html.hbs b/templates-untranslated/about/usage.html.hbs
index 8225180..337d5c4 100644
--- a/templates-untranslated/about/usage.html.hbs
+++ b/templates-untranslated/about/usage.html.hbs
@@ -1,5 +1,5 @@
 <div class="about usage">
-    <center><h2><a href="/about">About</a> | <a href="/about/news">News</a> | Usage | <a href="/about/faq">FAQ</a> | <a href="/about/stats">Stats</a> | <a href="/about/privacy">Privacy</a></h2></center>
+    <center><h2><a href="/about">About</a> | Usage | <a href="/about/faq">FAQ</a> | <a href="/about/privacy">Privacy</a></h2></center>

     <p>
         On this page, we collect information on how to use
