diff --git a/config/application.rb b/config/application.rb
index 07e0d6f38..a8cccc172 100644
--- a/config/application.rb
+++ b/config/application.rb
@@ -103,8 +103,10 @@ module Loomio
         user_name: ENV['SMTP_USERNAME'],
         password: ENV['SMTP_PASSWORD'],
         domain: ENV['SMTP_DOMAIN'],
-        ssl: ENV['SMTP_USE_SSL'],
-        openssl_verify_mode: 'none'
+        ssl: false, # only supporting STARTTLS for now
+        enable_starttls: true,
+        openssl_verify_mode: 'peer',
+        ca_file: "/etc/ssl/certs/ca-certificates.crt"
       }.compact
     else
       config.action_mailer.delivery_method = :test
